test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -630,51 +630,125 @@ def test_make_uri_prop(self) -> None:
         invalid_values = ["https:", 10.0, 5, "www.test.com"]
         run_test(self, prop, method, different_values, invalid_values)
 
-    def test_make_annotation_link_region(self) -> None:
-        """
-        This method tests 3 methods at the same time: make_annotation(), make_link(), and make_region().
-        """
-        method_2_tagname: dict[Callable[..., etree._Element], str] = {
-            excel2xml.make_annotation: "annotation",
-            excel2xml.make_link: "link",
-            excel2xml.make_region: "region",
-        }
-        for method, tagname in method_2_tagname.items():
-            test_cases: list[tuple[Callable[..., etree._Element], str]] = [
-                (
-                    lambda: method("label", "id"),
-                    f'<{tagname} label="label" id="id" permissions="res-default"/>',
-                ),
-                (
-                    lambda: method("label", "id", "res-restricted"),
-                    f'<{tagname} label="label" id="id" permissions="res-restricted"/>',
-                ),
-                (
-                    lambda: method("label", "id", ark="ark"),
-                    f'<{tagname} label="label" id="id" permissions="res-default" ark="ark"/>',
-                ),
-                (
-                    lambda: method("label", "id", iri="iri"),
-                    f'<{tagname} label="label" id="id" permissions="res-default" iri="iri"/>',
-                ),
-                (
-                    lambda: method("label", "id", creation_date="2019-10-23T13:45:12Z"),
-                    (
-                        f'<{tagname} label="label" id="id" permissions="res-default" '
-                        'creation_date="2019-10-23T13:45:12Z"/>'
-                    ),
-                ),
-            ]
-            for _method, result in test_cases:
-                xml_returned_as_element = _method()
-                xml_returned = etree.tostring(xml_returned_as_element, encoding="unicode")
-                xml_returned = regex.sub(r" xmlns(:.+?)?=\".+?\"", "", xml_returned)
-                self.assertEqual(result, xml_returned)
-
-            with self.assertWarns(UserWarning):
-                method("label", "id", ark="ark", iri="iri")
-            with self.assertRaisesRegex(BaseError, "invalid creation date"):
-                method("label", "restype", "id", creation_date="2019-10-23T13:45:12")
+    def test_make_annotation(self) -> None:
+        """Tests a regular annotation."""
```

Comment:
Now that we don't require docstrings in test files any more, you could also remove some of the superfluous docstrings.

Reply:
I meant the superfluous docstrings that you introduced in this PR :-)